contracts-periphery/.gas-snapshot

```diff
@@ -0,0 +1,154 @@
+BatchSendWithTollAndWithTokenBalanceGasTest:testPostSetupState() (gas: 11857)
```

Comment:
_(arbitrary comment location to start a thread)_

Nit: can we delete `fs_permissions = [{ access = "read", path = "./"}]` from the `profile.ci` section? If a key is undefined, it automatically uses the default value, so we don't need to redefine it twice

Reply:
gotcha